imageroot/pypkg/webtop.py

```diff
@@ -31,3 +32,8 @@ def configure_module(mail_module, penv):
         },
         "pecbridge_admin_mail": penv.get('PECBRIDGE_ADMIN_MAIL',''),
     })
+    agent.assert_exp(
+        configure_retval['exit_code'] == 0,
+        f"The configure-module subtask failed (exit_code={configure_retval['exit_code']})"
```

Comment:
The exit code of an action is already logged, there's no need to repeat it here.

```suggestion
        "The configure-module subtask failed"
```